README.md

```diff
@@ -42,7 +42,7 @@ The Zinc 1.0 incremental compiler implements significant improvements over
 0.13.13's version when it comes to performance, correctness and dependency
 analysis.
 
-Zinc 1.0 is already in use in many tools in the Scala ecosystem like sbt,
+Zinc 1.0 is already in use in many tools in the Scala ecosystem like [Mill](https://com-lihaoyi.github.io/mill/), sbt,
 Bloop, Pants, Bazel, and [Maven](https://github.com/random-maven/scalor-maven-plugin).
```

Comment:
sure, but maybe we should sort this by [frequency of usage](https://scalacenter.github.io/scala-developer-survey-2019/#which-build-tools-do-you-use) instead of listing Mill at top?

Reply:
My rule of thump for non-personal lists is alphabetical order. If you have at least three to four items, the alphabetical ordering is obvious to everyone and nobody needs to complain. Whereas any evidence-based ordering is typically time-bound and needs to be updated to stay fair.

Reply:
That said, I think sbt should probably keep the pole position as it is the origin of zinc. ;-)